src/go/rpk/pkg/config/config_test.go

```diff
@@ -470,6 +470,46 @@ func TestDefault(t *testing.T) {
 	require.Exactly(t, expected, defaultConfig)
 }
 
+func TestProdDefault(t *testing.T) {
+	defaultConfig := ProdDefault()
+	expected := &Config{
+		fileLocation:   DefaultPath,
+		Pandaproxy:     &Pandaproxy{},
+		SchemaRegistry: &SchemaRegistry{},
+		Redpanda: RedpandaNodeConfig{
+			Directory: "/var/lib/redpanda/data",
+			RPCServer: SocketAddress{"0.0.0.0", 33145},
+			KafkaAPI: []NamedAuthNSocketAddress{{
+				Address: "0.0.0.0",
+				Port:    9092,
+			}},
+			AdminAPI: []NamedSocketAddress{{
+				Address: "0.0.0.0",
+				Port:    9644,
+			}},
+			ID:            nil,
+			SeedServers:   []SeedServer{},
+			DeveloperMode: false,
+		},
+		Rpk: RpkConfig{
+			CoredumpDir:        "/var/lib/redpanda/coredump",
+			Overprovisioned:    false,
+			TuneAioEvents:      true,
```

Comment:
cc @RafalKorepta for awareness, these tuners will now be set to true

Reply:
> Operator doesn't call tune at all

This is good to know.